### PR TITLE
chore(tracking): Render tax info and link to help center for unlisted works + tracking events

### PR DESCRIPTION
**Pull request by jpotts244**
This PR resolves [AMBER-670](https://artsyproduct.atlassian.net/browse/AMBER-670)

### Description
Adds tax info and shipping details to unlisted artwork page

Adds in tracking event when tapped

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Adds Shipping and Tax info section to private artwork experience - Jackie

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Adds tracking event for tapping on Tax info learn more - Jackie

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[AMBER-670]: https://artsyproduct.atlassian.net/browse/AMBER-670?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Artwork/Components/ShippingAndTaxes.tsx [hunk 1]
```diff
@@ -49,6 +42,13 @@ const ShippingAndTaxes: React.FC<ShippingAndTaxesProps> = ({ artwork }) => {
           {priceIncludesTaxDisplay}
         </Text>
       )}
+
+      {!!taxInfo && (
```
Comment (jpotts244) [inline src/app/Scenes/Artwork/Components/ShippingAndTaxes.tsx:46]: Render tax info line to get closer to design parity

Before: 

Screenshot 2024-05-03 at 1 12 48 PM

---

After:

Screenshot 2024-05-03 at 12 50 39 PM

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 672a2c583da3bae2a7cd73a79aba8d46079240a2
</p>

File: src/app/Scenes/Artwork/Components/ShippingAndTaxes.tsx (hunk 1 above)
Comment (jpotts244) [reply src/app/Scenes/Artwork/Components/ShippingAndTaxes.tsx:58]: Added a tracking event to fire as per your document Louis!

One question for Louis, as currently written in the PR, this event will fire for anytime the Learn more link it tapped on any artwork page, is that ok? or should we only be firing on Private artwork page/context?

File: src/app/Scenes/Artwork/Components/ShippingAndTaxes.tests.tsx [hunk 2]
```diff
@@ -1,4 +1,7 @@
+import { fireEvent, screen } from "@testing-library/react-native"
```
Comment (jpotts244) [inline src/app/Scenes/Artwork/Components/ShippingAndTaxes.tests.tsx:1]: Heavy spec re-work to make the linter happy 👇🏻

**Review by damassi — APPROVED** [review]

**Pull request merged**